### ISSUE TITLE
Resume will override learning rate on command line.

### ISSUE THREAD

**Issue by manishbcd** [pytorch/examples#816]
If you resume from a checkpoint and want to run with a new learning rate, the saved learning rate will overwrite what you specify on the command line. The fix is simple.

**Comment by msaroufim** [comment]
Would you like to make the fix?
